App.js

```diff
@@ -4,9 +4,6 @@ import * as React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import LoginView from './components/LoginView/LoginView';
```

Comment:
Nice spot finding those unnecessary imports. Those were leftover from before we had the React Navigator installed.